Documentation~/modes.md

```diff
@@ -1,42 +1,20 @@
-# Edit modes (Object vs Element)
+# Edit modes and active contexts
 
-ProBuilder uses **Edit modes** to define what you are selecting and editing.
+There are two active contexts when editing in the Unity Editor:
 
-![Editing Modes Example](images/ExampleImage_ObjectAndElementEditingModes.png)
+* The GameObject context, which controls the entire Mesh.
```

Comment:
The GameObject context doesn't really control the mesh. *You* activate the context to be able to use transform tools on those meshes.

Reply:
```suggestion
* The GameObject context, which allows you to control the entire Mesh.
```